server.js

```diff
@@ -0,0 +1,117 @@
+const config = require('config');
+const fs = require('fs');
+const hapi = require('hapi');
+const HapiSwagger = require('hapi-swagger');
+const http2 = require('http2');
+const Inert = require('inert');
+const Vision = require('vision');
+const Routes = require('./api/index');
+
+/**
+ * Default environment configuration options
+ * WARNING! NOT ENVIRONMENT VARIABLES LIKE process.env.NODE_ENV
+ * If needed elsewhere, this will need to be redone
+ * @type {[type]}
+ */
+const env = config.get(process.env.NODE_ENV || 'development');
+
+/**
+ * TLS Configuration with Self Signed Certs
+ * allowHTTP1 is needed for Software that does not support
+ * HTTP2 like Postman :(
+ * @type {Object}
+ */
+const tls = {
+  key: fs.readFileSync('./sss-key.pem'),
+  cert: fs.readFileSync('./sss-cert.pem'),
+  allowHTTP1: true,
+};
+
+const listener = http2.createSecureServer(tls);
+
+/**
+ * Hapi Server configuration
+ * @type {[type]}
+ */
+const server = hapi.server({
+  host: env.HOST || 'localhost',
+  listener,
+  port: env.PORT || '3000',
+  router: {
+    stripTrailingSlash: true,
+  },
+  routes: {
+    cors: {
+      origin: env.ORIGIN || '*',
```

Comment:
\* should only be defaulted in Dev environment

Josh, 

What am I missing here in comparison to the old file service. Specifically the Allow-Methods headers